modules/installation/pages/client-requirements.adoc

```diff
@@ -23,15 +23,13 @@ For details on supported product versions, see https://www.suse.com/lifecycle.
 | {sle} 11 | {x86}, {x86}_64, {ipf}, IBM {ppc}, {ibmz}  | icon:check[role="green"]      | icon:check[role="green"]
 | {sls}-ES 7 | {x86}_64                                 | icon:check[role="green"]      | icon:check[role="green"]
 | {sls}-ES 6 | {x86}_64                                 | icon:check[role="green"]      | icon:check[role="green"]
-| {rhel} 8 | {x86}_64                                   | icon:question[role="gray"] | icon:question[role="gray"]
+| {sls} for SAP | {x86}_64                              | icon:check[role="green"]      | icon:check[role="green"]
```

Comment:
I see a lot of follow up questions comming...

We need to add versions similar to what we say for normal SLES and we need to check architectures.
For 12 and 15 we also support ppc64le as well. Not sure about version 11.

Reply:
Seems excessive to me, honestly. How deep does this rabbit hole go?

Reply:
Either we list all versions and architectures or we say explicit 'all' versions and 'all' architectures.
Otherwise we will get back questions. We tried already to list only some products and everytime we got bugs to add also this and that.

Reply:
*sigh*

OK, tell me what I need to add?

Reply:
Hmm, with all these macros used I lost a bit the idea how the whole table look like.

Now I had a look at the "resulting" page and saw that we talk about 

  "SUSE Linux Enterprise <VERSION>" 

which is pretty generic and include all kind of products. 
When we start to talk about "for SAP" specially we need to say also "Server", "Desktop", "HPC" and whatever comes next.

What about a footnote saying it includes the following products in the available architectures ... 
and list them there.

But now my feeling is we should discuss it also with PM.

Reply:
I've emailed Benajmin Landorf, @joesusecom, @paususe, and @mcalmer for advice.

Reply:
Bumped email 2019-07-01

Reply:
I haven't gotten any response to this at all. I'm going to merge and close the ticket and then we can see who yells :woman_shrugging:

Reply:
But please fix the architecture. SLES for SAP is also available for ppc64le.